posts/inside-rust/2020-03-09-rename-rustc-guide.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "The rustc-guide is now rustc-dev-guide"
+author: mark-i-m
+description: "the guide has been renamed"
+team: the rustc learning working group <https://www.rust-lang.org/governance/teams/compiler#wg-learning>
+---
+
+You may or may not be aware of two similarly named resources:
+- [The rustc book](https://doc.rust-lang.org/rustc/index.html)
+- [The rustc-guide](https://rust-lang.github.io/rustc-dev-guide/)
+
+What's difference? The "rustc book" is intended as a guide for _users_ of the
+compiler. The "rustc-guide" is intended as a guide for _developers_ of the
+compiler. However, the names have always been confusingly similar.
+
+**For that reason, we have decided to rename the `rustc-guide` to
+`rustc-dev-guide`.**
+
+We have put significant work into finding and updating links around the
+`rust-lang/*` repos. We have also put up a website in place of the former
+`rustc-guide` website that points to the book and rustc-dev-guide.
+
```

Comment:
Should we add a part suggesting to update remote link for local repos? Something like:

If you have a local clone of the rustc-dev-guide repository, we recommand that you update the remote URL to the GitHub repo:
```sh
git remote set-url origin https://github.com/rust-lang/rustc-dev-guide.git
```